akka-http-core/src/test/scala/akka/http/model/UriSpec.scala

```diff
@@ -406,7 +418,7 @@ class UriSpec extends WordSpec with Matchers {
       normalize("http://sourceforge.net/projects/uriparser/") shouldEqual "http://sourceforge.net/projects/uriparser/"
       normalize("http://sourceforge.net/project/platformdownload.php?group_id=182840") shouldEqual "http://sourceforge.net/project/platformdownload.php?group_id=182840"
       normalize("mailto:test@example.com") shouldEqual "mailto:test@example.com"
-      normalize("file:///bin/bash") shouldEqual "file:///bin/bash"
+      normalize("file:///bin/bash") shouldEqual "file:/bin/bash"
```

Comment:
URIs with a scheme but no authority are pretty rare, so looking for examples isn't as easy.
I'm not sure what the "scheme specification" for HTTP has to say about URIs with an empty authority. It's appears to be another unspeced edge case.
Semantically there doesn't appear to be a difference between `http:///foo` and `http:/foo`, so I'd day normalizing the first to the latter is ok.
Since in order to be able to tell these two variants apart we'd have to introduce another special case to our model I'm inclined to simply go with this simpler version and wait for someone to come up with a real-world case where this poses a problem. Then we can go back and decide whether the complication is indeed justified.